src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: $card-size;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  margin-right: $card-margin;
+  margin-bottom: 46px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__itemName {
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $nav-link-color;
+  }
+
+  &__productCode {
+    font-size: 10px;
+    margin-bottom: 16px;
+    font-weight: 400;
+    color: $card-text-color;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    line-height: 18px;
+  }
+
+  &__starReviews {
+    font-size: 10px;
+    margin: 0;
+    font-weight: 400;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__priceName {
+    font-size: 12px;
+    font-weight: 400;
+    color: $card-text-color;
+  }
+
+  &__priceNumber {
+    font-size: 16px;
+    font-weight: 600;
+    color: $nav-link-color;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    background-color: $card-button-color;
+    border-radius: 5px;
+    color: white;
+    border: 1px solid $card-button-color;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    transition: 0.3s;
```

Comment:
specify the properties for which you use transition
by default it'll be used to all properties, which can affect on website performance in bad way